### ISSUE TITLE
Split `admiral_qs` into `admiralophtha_qs` and `admiral_qs`

### ISSUE THREAD

**Issue by manciniedoardo** [pharmaverse/admiraldata#6]
### Background Information

After the completion of #1 we should have the QS dataset from {admiral.test} available in {admiraldata}. This dataset had been originally edited in {admiral.test} to include VFQ parameters upon request from the {admiralophtha} team (see the update_qs script). Now, it would be best to create two copies of the `QS` SDTM: `admiralophtha_qs`, which contains everything, and `admiral_qs`, which does not contain VFQ records. 

Note: the {admiralophtha} team has made an update to how the VFQ records are derived (see [here](https://github.com/pharmaverse/admiralophtha/blob/main/dev/vfq.R) for the updated script) so this code should be used instead.

### Definition of Done

`admiral_qs` and `admiralophtha_qs` as described above present in the `data` folder and documented in `data.R`.

**Comment by bundfussr** [comment]
As far as I know `admiral_qs` is not used. The questionnaires vignette uses `example_qs` stored in admiral.
Should we move `example_qs` to admiraldata?
Should we remove `admiral_qs`?

**Comment by bms63** [comment]
This is a huge datasets and I would say please remove it!!

**Comment by kaz462** [comment]
Agreed in the meeting: remove `admiral_qs`, keep `admiralophtha_qs` and `example_qs` from questionnaires vignette in {admiraldata}

`example_qs` has different subject IDs, a clean-up will be needed in a later phase.
